packages/fluentui/docs/src/examples/components/Table/Usage/TableExampleNavigable.shorthand.tsx

```diff
@@ -50,11 +52,14 @@ const rowsPlain = [
       { content: 'Roman van von der Longername', key: '1-2', id: 'name-1' },
       { content: 'None', key: '1-3' },
       { content: '30 years', key: '1-4', id: 'age-1' },
-      moreActionCell,
-      moreOptionCell
+      moreActionCell('1-5'),
+      moreOptionCell('1-6')
```

Comment:
I would prefer an approach with spread:

```suggestion
      { key: '1-6', ...moreOptionCell },
```

But it's not blocking

Reply:
fixed :)